### PR TITLE
Docs endpoint header basics

### LINKED ISSUE #1560: Implement docs > linodes > endpoint > header component
This represents a docs endpoint page header, includes:

- url
- methods
- description
etc

### PR DESCRIPTION
**Pull request by mparke**
closes #1560 

- Fixed some issues with routing
- Improved crumbs ( still needs changes )
- Add basic endpoint/method/example components

Example:

screen shot 2017-04-24 at 5 35 40 pm

### CODE REVIEW

**Comment by eatonphil** [comment]
The breadcrumbs for this page should be the sections it appears under, not the urls. So here it would say "Linodes > Backups".

File: docs/scss/components/Endpoint.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        padding: 0;
+
+        li {
+            display: inline;
+            margin-left: 10px;
+        }
+    }
+
+    .divider {
+     border: $border;
```
Comment (eatonphil) [inline docs/scss/components/Endpoint.scss]: Spacing?

**Comment by eatonphil** [comment]
Maybe add an issue to make the tabbed example component?

**Comment by eatonphil** [comment]
Also maybe add an issue to break endpoints out into groups? e.g. Linodes into Instances, Backups, Disks, etc.

**Comment by mparke** [comment]
Follow ups:

https://github.com/linode/manager/issues/1636
https://github.com/linode/manager/issues/1637

**Pull request merged**